### DISCUSSION TITLE
Stalled build

### DISCUSSION THREAD

**Discussion by 1800joe** [noop-inc/documentation#83] [Bugs] [answered]
It seems like this build is stalled:

<img width="957" alt="Screenshot 2024-01-23 at 9 45 23 AM" src="https://github.com/noop-inc/documentation/assets/3111670/aac8d58d-720b-4b15-85ea-93c582ae5f7c">

The Environment can't be killed in this state.

**Comment by nathanielks** [comment]
I'm also experiencing this on a separate application. It's been 25 minutes since the last log event was emitted.

![Screenshot 2024-01-23 at 12 54 44 PM](https://github.com/noop-inc/documentation/assets/314009/9967e1d1-ee9b-4c2d-8e49-e1039c5942fb)

**Reply by nathanielks** [reply]
Of note in case it's important, I had ~3 environments running at the time and memory consumption on the lima instance was around ~6GB

**Comment by 1800joe** [comment, marked as answer]
This appears resolved as of v0.7.0
